cfme/tests/openstack/cloud/test_volumes.py

```diff
@@ -110,3 +120,31 @@ def test_edit_volume_with_type(volume_with_type, appliance):
 def test_delete_volume_with_type(volume_with_type):
     volume_with_type.delete()
     assert not volume_with_type.exists
+
+
+@pytest.mark.regression
+def test_volume_attach_detach_instance(volume, new_instance, appliance):
```

Comment:
I would like to see `Reset` and `Cancel` tested as well, may be out of scope for this PR. Add a `TODO` perhaps?

Reply:
I think also it is out of scope for this PR. Will add a TODO